AnkiDroid/src/main/java/com/ichi2/compat/CompatV30.java

```diff
@@ -0,0 +1,40 @@
+
+
+package com.ichi2.compat;
+
+
+
+import android.annotation.TargetApi;
+
+import android.view.View;
+import android.view.Window;
+import android.view.WindowInsets;
+import android.view.WindowInsetsController;
+
+import com.ichi2.anki.R;
+import com.ichi2.anki.reviewer.FullScreenMode;
+
+
+/** Implementation of {@link Compat} for SDK level 30 and higher. Check  {@link Compat}'s for more detail. */
```

Comment:
I love this comment, great!

```suggestion
/** Implementation of {@link Compat} for SDK level 30 and higher. Check {@link Compat}'s for more detail. */
```